app/views/card_grants/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<% title "Card grants for #{@event.name}" %>
+<% page_md %>
+<%= render "events/nav", selected: :card_grants %>
+<%= render "card_grants/create_modal" %>
+
+<h1 class="flex heading items-center">
+  <span class="flex-grow flex items-center flex-1">
+    Card grants
+    <%= badge_for @event.card_grants.active.count, class: "bg-muted" %>
+  </span>
+
+  <% if organizer_signed_in?(as: :member) %>
+    <%= link_to new_event_card_grant_path(@event), class: "btn bg-success", data: { behavior: "modal_trigger", modal: "create_card_grant" }, disabled: !policy(@event).create_transfer? do %>
+      <%= inline_icon "card-add" %>
+      Send a grant
+    <% end %>
+  <% end %>
+</h1>
+
+<%= turbo_frame_tag :card_grant_card_overview, src: event_card_grant_card_overview_path(@event), loading: :eager do %>
+  <p>Loading card grants</p>
+<% end %>
+
+<%= turbo_frame_tag :card_grant_transaction_overview, src: event_card_grant_transaction_overview_path(@event), loading: :eager do %>
+  <p>Loading transactions</p>
+<% end %>
```

Comment:
nit: loading state could look better but it's fine for now <img width="958" height="306" alt="image" src="https://github.com/user-attachments/assets/fe0de8c6-2dda-4d1d-9db4-143f1be7ae99" />